app/src/main/java/com/willowtree/vocable/settings/EditCategoriesListFragment.kt

```diff
@@ -161,42 +152,39 @@ class EditCategoriesListFragment : BaseFragment<FragmentEditCategoriesListBindin
             }
 
             if (category.categoryId == PresetCategories.USER_FAVORITES.id) {
-                editCategorySelectButton.isEnabled = false
+                editCategorySelectButton?.isEnabled = false
             }
 
-            with(editCategorySelectButton) {
-                // enable the button if the category is user-generated, or if it's My Sayings
-                isEnabled =
-                    category.isUserGenerated || category.categoryId == PresetCategories.USER_FAVORITES.id
-
-                action = {
-                    val action =
-                        EditCategoriesFragmentDirections.actionEditCategoriesFragmentToEditCategoryOptionsFragment(
-                            category
-                        )
-                    if (findNavController().currentDestination?.id == R.id.editCategoriesFragment) {
-                        findNavController().navigate(action)
-                    }
-                    editCategoriesViewModel.onCategorySelected(category)
+            individualEditCategoryButton?.action = {
+                val action =
+                    com.willowtree.vocable.settings.EditCategoriesFragmentDirections.actionEditCategoriesFragmentToEditCategoryOptionsFragment(
+                        category
+                    )
+                if (findNavController().currentDestination?.id == com.willowtree.vocable.R.id.editCategoriesFragment) {
+                    findNavController().navigate(action)
                 }
+                editCategoriesViewModel.onCategorySelected(category)
             }
-        }
 
-        with(editButtonBinding.showHideCategoryButton) {
-            if (!category.hidden) {
-                setImageResource(R.drawable.button_hidden)
-                setBackgroundResource(R.drawable.button_default_background)
-            } else {
-                setImageResource(R.drawable.button_shown)
-                setBackgroundResource(R.drawable.category_button_background)
-            }
+/* Commenting this out so I can keep this code for when we refactor the category editing screen
```

Comment:
I confused why we'll need this?

Reply:
There is this "show category" functionality that I was assuming replace the "hide" functionality...But maybe this is something we double check with Sashank in our check-in to see how that works. Because I'm not seeing from the designs how the "show category" should work and if it should replicate the "hide" functionality. 


<img width="513" alt="Screen Shot 2022-04-26 at 2 05 28 PM" src="https://user-images.githubusercontent.com/14029158/165363934-b3e8646e-91f9-4f12-9f88-3b8635ca4361.png">